### PR TITLE
Worksheets 72-78

### PR DESCRIPTION
**Pull request by longhotsummer**
Missing worksheets 72, 74
Aggregates and key themes for 75-78

### CODE REVIEW

**Comment by longhotsummer** [comment]
@xybrnet

File: reports/report_builder.py [hunk 1]
```diff
@@ -1759,6 +1759,62 @@ def ws_68b(self, ws):
...
+        """
+        Cols: Focus Topic, Media
+        Rows: Country
+        Focus: female reporters
+        """
+        # TODO: these values should be %age total of media in country/region
+
+        secondary_counts = OrderedDict()
+        for topic_id, topic in FOCUS_TOPICS.iteritems():
+            actual_topic_ids = [k for k, v in FOCUS_TOPIC_MAP.iteritems() if v == topic_id]
```
Comment (xybrnet) [inline reports/report_builder.py]: Should this not be the following?
`actual_topic_ids = [v for k, v in FOCUS_TOPIC_MAP.iteritems() if k == topic_id]`

Comment (longhotsummer) [inline reports/report_builder.py]: I've reversed the map so that it's clearer what's going on. We weren't using this map effectively anyway.

Comment (xybrnet) [inline reports/report_builder.py]: Ah sorry, I missed that.

**Pull request merged**